.github/workflows/create-dev-tag.yml

```diff
@@ -52,15 +52,25 @@ jobs:
           git_push_gpgsign: false
 
       - name: Generate tag
-        run: echo "TAG=$(make ci.docker.run RUN_WHAT="make repo.dev.tag" | tail -1)" >> $GITHUB_ENV
+        run: echo "TAG=$(make repo.dev.tag | tail -1)" >> $GITHUB_ENV
         env:
           GITHUB_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
 
       - name: Create tag
-        run: git tag -s -f -m "${{ env.TAG }}" ${{ env.TAG }}
+        run: |
+          ## Committer email does not match GPG private key email
+          ## So we need to set it expliclty here
+          git config user.email "ci-mergebot@d2iq.com"
+          git config user.name "ci-mergebot"
```

Comment:
Not sure why this needs to be done once here and again for global below? As mentioned though, I think we can just use the gpg import action to do this.